linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml

```diff
@@ -61,17 +65,39 @@ warnings:
         this process.
 
 fixtext: |-
-    Configure {{{ full_name }}} to implement DoD-approved encryption by following the steps below:
-
-    To enable strict FIPS compliance, the fips=1 kernel option needs to be added to the kernel boot
-    parameters during system installation so key generation is done with FIPS-approved algorithms
-    and continuous monitoring tests in place.
-
-    Enable FIPS mode after installation (not strict FIPS compliant) with the following command:
+    Configure the operating system to implement FIPS mode with the following command
 
     $ sudo fips-mode-setup --enable
 
     Reboot the system for the changes to take effect.
 
 srg_requirement:
-    '{{{ full_name }}} must implement NIST FIPS-validated cryptography for the following: to provision digital signatures, to generate cryptographic hashes, and to protect data requiring data-at-rest protections in accordance with applicable federal laws, Executive Orders, directives, policies, regulations, and standards.'
+    '{{{ full_name }}} must implement NIST FIPS-validated cryptography for the following: to
+    provision digital signatures, to generate cryptographic hashes, and to
+    protect data requiring data-at-rest protections in accordance with
+    applicable federal laws, Executive Orders, directives, policies,
+    regulations, and standards.'
+
+vuldiscussion: |-
+    If maintenance tools are used by unauthorized personnel, they may accidentally or intentionally
```

Comment:
Please remove this does not match the Vul Discussion in the spreadsheet.

Reply:
Hmm, it does match the spreadsheet, but `SRG VulDiscussion`, not `Vul Discussion`.

Now I'm confused, does the `SRG VulDiscussion` matter?

Reply:
The `SRG VulDiscussion` is a static field used during the development of a STIG. We want the `Vul Discussion` which is specific to our product.